common/types.go

```diff
@@ -652,7 +1067,7 @@ func (loc Location) CommonDom(cmp Location) Location {
 }
 
 func (l Location) ContainsAddress(a Address) bool {
-	prefix := a[0]
+	prefix := a.inner.Bytes()[0]
```

Comment:
(nit) don't need to reference the `inner` here. Should just use the `Bytes()` interface directly

Reply:
Fixed